src/ints/int10_modes.cpp

```diff
@@ -1949,51 +2056,67 @@ bool INT10_SetVideoMode(uint16_t mode)
 		IO_WriteB(crtc_base,0x50); IO_WriteB(crtc_base+1,reg_50);
 
 		uint8_t reg_31, reg_3a;
+
 		switch (CurMode->type) {
-			case M_LIN15:
-			case M_LIN16:
-			case M_LIN24:
-			case M_LIN32:
-				reg_3a=0x15;
-				break;
-			case M_LIN8:
-				// S3VBE20 does it this way. The other double pixel bit does not
-				// seem to have an effect on the Trio64.
-			        if (CurMode->special & VGA_PIXEL_DOUBLE)
-				        reg_3a = 0x5;
-			        else
-				        reg_3a = 0x15;
-			        break;
-		        default: reg_3a = 5; break;
-		        };
+		case M_LIN15:
+		case M_LIN16:
+		case M_LIN24:
+		case M_LIN32: reg_3a = 0x15; break;
+
+		case M_LIN8:
+			// S3VBE20 does it this way. The other double pixel bit
+			// does not seem to have an effect on the Trio64.
+			if (CurMode->special & VGA_PIXEL_DOUBLE) {
+				reg_3a = 0x5;
+			} else {
+				reg_3a = 0x15;
+			}
+			break;
+		default: reg_3a = 5; break;
+		};
 
 		switch (CurMode->type) {
-		case M_LIN4: // <- Theres a discrepance with real hardware on this
+		case M_LIN4:
+			// Theres a discrepancy with real hardware on this
 		case M_LIN8:
 		case M_LIN15:
 		case M_LIN16:
 		case M_LIN24:
-		case M_LIN32:
-			reg_31 = 9;
-			break;
-		default:
-			reg_31 = 5;
-			break;
+		case M_LIN32: reg_31 = 9; break;
+		default: reg_31 = 5; break;
 		}
-		IO_Write(crtc_base,0x3a);IO_Write(crtc_base+1,reg_3a);
-		IO_Write(crtc_base,0x31);IO_Write(crtc_base+1,reg_31);	//Enable banked memory and 256k+ access
-		IO_Write(crtc_base,0x58);IO_Write(crtc_base+1,0x3);		//Enable 8 mb of linear addressing
 
-		IO_Write(crtc_base,0x38);IO_Write(crtc_base+1,0x48);	//Register lock 1
-		IO_Write(crtc_base,0x39);IO_Write(crtc_base+1,0xa5);	//Register lock 2
+		IO_Write(crtc_base,0x3a);
+		IO_Write(crtc_base+1,reg_3a);
+
+		IO_Write(crtc_base,0x31);
+		// Enable banked memory and 256k+ access
+		IO_Write(crtc_base+1,reg_31);
+
+		IO_Write(crtc_base,0x58);
+		// Enable 8 mb of linear addressing
+		IO_Write(crtc_base+1,0x3);
+
+		IO_Write(crtc_base,0x38);
+		// Register lock 1
+		IO_Write(crtc_base+1,0x48);
+
+		IO_Write(crtc_base,0x39);
+		// Register lock 2
+		IO_Write(crtc_base+1,0xa5);
```

Comment:
I'll give you a hand. 😄 
```suggestion
		IO_Write(crtc_base,0x3a);
		IO_Write(crtc_base+1,reg_3a);

		// Enable banked memory and 256k+ access
		IO_Write(crtc_base,0x31);
		IO_Write(crtc_base+1,reg_31);

		// Enable 8 mb of linear addressing
		IO_Write(crtc_base,0x58);
		IO_Write(crtc_base+1,0x3);

		// Register lock 1
		IO_Write(crtc_base,0x38);
		IO_Write(crtc_base+1,0x48);

		// Register lock 2
		IO_Write(crtc_base,0x39);
		IO_Write(crtc_base+1,0xa5);
```

Reply:
Thanks @japsmits 😄